Makefile

```diff
@@ -1,59 +1,149 @@
+# Project variables
 NAME        := terraform-docs
 VENDOR      := segmentio
 DESCRIPTION := Generate docs from Terraform modules
 MAINTAINER  := Martin Etmajer <metmajer@getcloudnative.io>
 URL         := https://github.com/$(VENDOR)/$(NAME)
 LICENSE     := MIT
 
-VERSION     := $(shell cat ./VERSION)
+# Repository variables
+PACKAGE     := github.com/$(VENDOR)/$(NAME)
```

Comment:
Since we now have ˋPACKAGEˋ and ˋURLˋ has never been used, let‘s remove the latter!

Reply:
`URL` is not used but it's kind of part of the metadata of the repository, the same as `LICENSE`. I prefer to keep it, and may be even not using any variables in it.